src/Extension/OrderedButtonExtension.php

```diff
@@ -24,6 +24,6 @@ class OrderedButtonExtension extends AbstractOrderedExtension
     */
     public function getExtendedType()
     {
-        return 'button';
+        return 'Symfony\Component\Form\Extension\Core\Type\ButtonType';
```

Comment:
You can add a check if the `getBlockPrefix()` method is present in the `AbstractType` class from the Form component and then either return the FQCN or the legacy name.